pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -137,7 +138,10 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 		return fmt.Errorf("couldn't create deployment for deployment config %s: %v", deployutil.LabelForDeploymentConfig(config), err)
 	}
 	c.recorder.Eventf(config, "DeploymentCreated", "Created new deployment %q for version %d", created.Name, config.LatestVersion)
-	return nil
+	// Update deploymentConfig status; we probably need a separate call for status
```

Comment:
The deployer pod updates the status on the replication controller, it never touches the config. I think that `status` would be internal only, @ironcladlou thoughts?